released/packages/coq-jsast/coq-jsast.2.0.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+name: "coq-jsast"
+version: "2.0.0"
```

Comment:
```suggestion
```
Name and version are already given by opam file path.

Reply:
Removed.